pytest.ini

```diff
@@ -11,3 +11,7 @@ addopts =
     --durations 10
     --durations-min 1
     --strict-markers
+    --cov
+    --cov-report html
+    --cov-report term
```

Comment:
@samdoran I believe you may want to have `term:skip-missing` instead.

Reply:
I think it would be `term:skip-covered` or `term-missing:skip-covered`. I have `skip_covered = True` in the `coveragerc` file and I don't think the output from missing lines is useful.